src/Sensor_TEST.cc

```diff
@@ -336,3 +336,159 @@ TEST(DOMSensor, EnableMetrics)
   sensor.SetEnableMetrics(false);
   EXPECT_EQ(false, sensor.EnableMetrics());
 }
+
+/////////////////////////////////////////////////
+TEST(DOMSensor, MutableSensors)
+{
+  // Altimeter
+  {
+    sdf::Sensor sensor;
+    sensor.SetType(sdf::SensorType::ALTIMETER);
+
+    sdf::Altimeter alt;
+    sensor.SetAltimeterSensor(alt);
+
+    sdf::Altimeter *altMutable = sensor.AltimeterSensor();
+    EXPECT_DOUBLE_EQ(altMutable->VerticalPositionNoise().Mean(),
+        sensor.AltimeterSensor()->VerticalPositionNoise().Mean());
+
+    sdf::Noise noise;
+    noise.SetMean(2.0);
+    altMutable->SetVerticalPositionNoise(noise);
+    EXPECT_DOUBLE_EQ(altMutable->VerticalPositionNoise().Mean(), 2.0);
+    EXPECT_DOUBLE_EQ(
+        sensor.AltimeterSensor()->VerticalPositionNoise().Mean(), 2.0);
```

Comment:
I think I'm doing that, if I understand your request. See line 352-353 for this sensor.

Reply:
Sorry for the confusion - on lines 352-353, the mean is checked to ensure a match between the mutable reference and `sensor.AltimeterSensor`, but there's no check of what the value of the mean is at that point (perhaps it's `0.0`?). Then, the mean is updated, and checked in lines 358-360 to make sure that the mean is now `2.0`. So, for the case of this sensor, I think it would be worth adding a check before line 356 to make sure that the mean isn't already `2.0`. That way, we can see that the mean was actually changed. Does that make sense? If so, then the same check applies to the other sensor tests in this PR

Reply:
The default mean value is 0.0, so line 352-353 cover this case.

Reply:
I'm also only trying to test that I can get mutable point versions. Checking whether noise is set properly and/or has proper default values is outside the scope of this PR.